pennylane/devices/default_clifford.py

```diff
@@ -0,0 +1,644 @@
+
+        The device represents a state by the inverse of a Tableau
+        (`Aaronson & Gottesman (2004) <https://journals.aps.org/pra/abstract/10.1103/PhysRevA.70.052328>`_)
+        consisiting of binary variable :math:`x_{ij},\ z_{ij}` for all :math:`i\in\left\{1,\ldots,2n\right\}`,
+        :math:`j\in\left\{1,\ldots,n\right\}`, and :math:`r_{i}` for all
+        :math:`i\in\left\{1,\ldots,2n\right\}`.
+
+        .. math::
+
+            \begin{bmatrix}
+            x_{11} & \cdots & x_{1n} &        & z_{11} & \cdots & z_{1n} & &r_{1}\\
+            \vdots & \ddots & \vdots & & \vdots & \ddots & \vdots & &\vdots\\
+            x_{n1} & \cdots & x_{nn} &        & z_{n1} & \cdots & z_{nn} & &r_{n}\\
+            & & & & & & & & \\
+            x_{\left(  n+1\right)  1} & \cdots & x_{\left(  n+1\right)  n} & &
+            z_{\left(  n+1\right)  1} & \cdots & z_{\left(  n+1\right)  n} & & r_{n+1}\\
+            \vdots & \ddots & \vdots  & & \vdots & \ddots & \vdots & & \vdots\\
+            x_{\left(  2n\right)  1}  & \cdots & x_{\left(  2n\right)  n} & &
+            z_{\left(  2n\right)  1}  & \cdots & z_{\left(  2n\right)  n} & & r_{2n}
+            \end{bmatrix}
+
+        Rows :math:`1` to :math:`n` of the tableau represent the destabilizer generators
+        :math:`R_{1},\ldots,R_{n}`, and rows :math:`n+1` to :math:`2n` represent the stabilizer
+        generators :math:`R_{n+1},\ldots,R_{2n}`. If :math:`R_{i}=\pm P_{1}\ldots P_{n}`,
+        then bits :math:`x_{ij},z_{ij}` determine the j\ :math:`^{th}` Pauli matrix
+        :math:`P_{j}:\ 00` means ``I``, :math:`01` means ``X``, :math:`11` means ``Y``,
+        and :math:`10` means ``Z``. Finally, :math:`r_{i}` is one if :math:`R_{i}`
+        has negative phase and zero if :math:`r_{i}` has a positive phase.
+
+
+    .. details::
+        :title: Tracking
+        :href: clifford-tracking
+
+        ``DefaultClifford`` tracks:
+
+        * ``executions``: the number of unique circuits that would be required on quantum hardware
+        * ``shots``: the number of shots
+        * ``resources``: the :class:`~.resource.Resources` for the executed circuit.
+        * ``simulations``: the number of simulations performed. One simulation can cover multiple QPU executions,
+          such as for non-commuting measurements and batched parameters.
+        * ``batches``: The number of times :meth:`~.execute` is called.
+        * ``results``: The results of each call of :meth:`~.execute`
+        * ``derivative_batches``: How many times :meth:`~.compute_derivatives` is called.
+        * ``execute_and_derivative_batches``: How many times :meth:`~.execute_and_compute_derivatives` is called
+        * ``derivatives``: How many circuits are submitted to :meth:`~.compute_derivatives` or
+          :meth:`~.execute_and_compute_derivatives`.
+
+    .. details::
+        :title: Accelerate calculations with multiprocessing
+        :href: clifford-multiprocessing
```

Comment:
If this is the same text and underlying functionality as default.qubit, can we just have a sentence pointing to DQ's docs section?

Reply:
Added a sentence. Currently linking to the class, but not sure how to directly link the section within it with sphinx magic.